modules/test/playwright/tests/frontend-data-set-views-web/tests/data-set-fragment/fixtures/fdsFragmentPageTest.ts

```diff
@@ -3,9 +3,15 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import {test} from '@playwright/test';
+import {mergeTests} from '@playwright/test';
 
 import {FDSFragmentPage} from '../pages/FDSFragmentPage';
+import {
+	dataSetManagerSetupTest
+} from "../../data-set-manager/fixtures/dataSetManagerSetupTest";
+
+// ensures data-set-fragment tests can run before data-set-admin tests
+const test = mergeTests(dataSetManagerSetupTest);
```

Comment:
@markocikos this is temporary. Once we move to the batch framework to create data model, we will be able to remove this. In the meantime we need data model there even if data-set-admin family of tests did not yet run